spec/index.bs

```diff
@@ -5885,9 +5885,17 @@ details.
                 [=interpolation=]
                 of the input.
             - There is less than |device|.limits.{{supported limits/maxInterStageShaderComponents}}
-                components of user-defined outputs for |descriptor|.{{GPURenderPipelineDescriptor/vertex}}.
+                components of user-defined and built-in outputs for |descriptor|.{{GPURenderPipelineDescriptor/vertex}}.
+              - If |descriptor|.{{GPURenderPipelineDescriptor/primitive}}.{{GPUPrimitiveState/topology}} is
+                  {{GPUPrimitiveTopology/"point-list"}}:
+                - There is less than (|device|.limits.{{supported limits/maxInterStageShaderComponents}} - 1)
+                    components of user-defined and built-in outputs for |descriptor|.{{GPURenderPipelineDescriptor/vertex}}.
             - There is less than |device|.limits.{{supported limits/maxInterStageShaderComponents}}
-                components of user-defined inputs for |descriptor|.{{GPURenderPipelineDescriptor/fragment}}.
+                components of user-defined and built-in inputs for |descriptor|.{{GPURenderPipelineDescriptor/fragment}}.
+            - The [=location=] of each user-defined output of |descriptor|.{{GPURenderPipelineDescriptor/vertex}} must
+                be less than (|device|.limits.{{supported limits/maxInterStageShaderComponents}} / 4).
```

Comment:
Maybe -1 to reserve for the `@builtin(position)`? Ditto for the fragment shader.